workbench/src/renderer/components/SetupTab/index.jsx

```diff
@@ -274,8 +275,20 @@ class SetupTab extends React.Component {
   }
 
   async loadParametersFromFile(filepath) {
-    const datastack = await fetchDatastackFromFile(filepath);
     const { pyModuleName, switchTabs, t } = this.props;
+    let datastack;
+    try {
+      datastack = await fetchDatastackFromFile(filepath);
+    } catch (error) {
+      logger.error(error);
+      alert( // eslint-disable-line no-alert
+        t(
+          'No InVEST model data can be parsed from the file:\n {{filepath}}',

```

Comment:
I wonder if this message could provide any more information to help the user solve the problem. Maybe something like 
> File could not be opened as an InVEST model run. Are you sure this is a valid datastack for a current InVEST model?

Reply:
I agree it's not perfect but I'm kind of at a loss for how to improve it. This is from the "Load Parameters" button on a model's sidebar, so we're not really opening a "model run" here. If "datastack" had a clear meaning to users then I'd agree we could say, "this is not a valid datastack", but I'm afraid that term is mostly insider-jargon. 

My hope is that anyone who sees this message will just realize they clicked on the wrong file or something. If they really don't know what file they should be looking for, then that's a larger design flaw and we should try to add some guidance before the user reaches this alert. We have a tooltip that tries to do that now.

Reply:
Oh I see what you mean. Yeah, this should be fine for now.